ovos_plugin_manager/templates/tts.py

```diff
@@ -154,19 +153,15 @@ def clear_queue(self):
         except Exception:
             pass
 
-    def begin_audio(self):
+    def begin_audio(self, message=None):
         """Perform beginning of speech actions."""
-        # This check will clear the "signal", in case it is still there for some reasons
-        check_for_signal("isSpeaking")
-        # this will create it again
-        create_signal("isSpeaking")
-        # Create signals informing start of speech
         if self.bus:
-            self.bus.emit(Message("recognizer_loop:audio_output_start"))
+            message = message or Message("speak")
```

Comment:
Would `dig_for_message` handle any calls where a message exists but wasn't passed?

Reply:
it's done in the TTS class execute method, this is only here for compat in case someone is importing the PlaybackThread directly. inside the PlaybackThread dig_for_message might not be accurate since the data here is async and comes from a queue